driver-sync/src/test/functional/com/mongodb/client/AbstractTransactionsTest.java

```diff
@@ -71,9 +64,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assume.assumeTrue;
 
-// See https://github.com/mongodb/specifications/tree/master/source/transactions/tests
-@RunWith(Parameterized.class)
-public class TransactionsTest {
+public class AbstractTransactionsTest {
```

Comment:
Now a base class for the two sets of transaction tests, so that the names of each are easily distinguishable.